src/xenia/base/memory_posix.cc

```diff
@@ -82,16 +88,19 @@ FileMappingHandle CreateFileMappingHandle(const std::filesystem::path& path,
   }
 
   oflag |= O_CREAT;
-  int ret = shm_open(path.c_str(), oflag, 0777);
+  auto full_path = "/" / path;
+  int ret = shm_open(full_path.c_str(), oflag, 0777);
   if (ret > 0) {
     ftruncate64(ret, length);
   }
 
   return ret <= 0 ? nullptr : reinterpret_cast<FileMappingHandle>(ret);
```

Comment:
0 is a valid handle value, probably should return a bool rather than the handle itself, but need to check where it's used.